contributors/design-proposals/api-machinery/customresources-validation.md

```diff
@@ -381,6 +381,28 @@ Note:
 
 2. For migration of CRDs with no validation to CRDs with validation, we can create a controller that will validate and annotate invalid CRs once the spec changes, so that the custom controller can choose to delete them (this is also essentially the status condition of the CRD). This can be achieved, but it is not part of the proposal.
 
+### Admission Webhook
+
+Custom resource definitions use the normal REST endpoint implementation and only customizes the registry and the codecs consequentaly dynamic web-hook admission controllers can be used
```

Comment:
Reword, hard to read-- this is missing a comma or something.

Reply:
Done.